packages/relay/tests/lib/eth.spec.ts

```diff
@@ -935,6 +937,27 @@ describe('Eth calls using MirrorNode', async function () {
     expect(firstFeeHistory).to.equal(secondFeeHistory);
   });
 
+  it('eth_estimateGas contract call returns default', async function() {
+    const gas = await ethImpl.estimateGas({input:"0x01"}, null);
```

Comment:
Your test will fail since you're giving a valid value to transaction which is `{input:"0x01"}`
As a result the first 2 checks you make will be
!transaction => false
!transaction.data => true
hence the non default gas value is used

Reply:
fixed